runtime/runtime/src/state_viewer.rs

```diff
@@ -116,6 +116,7 @@ impl TrieViewer {
                 predecessor_account_id: originator_id.clone(),
                 input: args.to_owned(),
                 block_index,
+                block_timestamp: 0,
```

Comment:
I don't think this is a good idea if the function being called relies on block stamp in some way.

Reply:
Had to change some trait signatures to fix that.